### PR TITLE
fix: buttons will not jump to the top when keyboard is used

### PR DESCRIPTION
**Pull request by AdityaHarvi**
#### Summary of changes/features added:
- Buttons will not jump to the top when the keyboard is used.
This is just a safety precaution.

#### Screenshots of UI changes (update with each new commit): 
- n/a

#### Link to referenced github issue: 
n/a

## Pull Request Checklist
- [ ] Unit test written?
- [ ] (**For later in project**) Integration tests written?
- [ ] Java docs written?

### CODE REVIEW

**Review by shadowciaw — APPROVED** [review]

**Comment by ducklin5** [comment]
lgtm, just resolve conflicts then I will approve

**Comment by AdityaHarvi** [comment]
I ended up deleting the MyProfileActivity.java file cause we aren't using it

File: client/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,20 +20,28 @@
...
-        <activity android:name=".Activities.NonOwnerBookViewActivity"/>
-        <activity android:name=".Activities.LoginActivity">
+        <activity android:name=".Activities.ScanActivity"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.SomethingWentWrong"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.CameraActivity"
+            android:windowSoftInputMode="adjustPan"/>
+        <activity android:name=".Activities.NonOwnerBookViewActivity"
+            android:windowSoftInputMode="adjustPan"/>
```
Comment (shadowciaw) [inline client/app/src/main/AndroidManifest.xml:30]: there's no keyboard input in any of the above activities fyi, up to you if you want to remove them

**Review by shadowciaw — APPROVED** [review]

**Review by ducklin5 — APPROVED** [review]
lgtm 🚀

**Pull request merged**